entgql/extension.go

```diff
@@ -378,72 +356,39 @@ func (e *Extension) whereExists() (int, bool) {
 }
 
 // updateSchema commits the changes to the GraphQL schema file.
-func (e *Extension) updateSchema(inputs map[string]*ast.InputObjectDefinition) error {
-	visitor.Visit(e.doc, &visitor.VisitorOptions{
-		LeaveKindMap: map[string]visitor.VisitFunc{
-			kinds.InputObjectDefinition: func(p visitor.VisitFuncParams) (string, interface{}) {
-				// If the input object was found in the schema, we update its definition.
-				if node, ok := p.Node.(*ast.InputObjectDefinition); ok && inputs[node.Name.Value] != nil {
-					input := inputs[node.Name.Value]
-					delete(inputs, node.Name.Value)
-					return visitor.ActionUpdate, input
-				}
-				return visitor.ActionNoChange, nil
-			},
-		},
-	}, nil)
```

Comment:
This's causes breaking changes. We can restore the old behavior but it required to parse the `ent.gql` file with `vektah/gqlparser`

Reply:
This logic was used to allow extending an existing GraphQL schema with the generated `WhereInput`s. Do we want to drop this support? We don't use the "upsert" option. 

cc @crossworth

Reply:
As I mentioned in [this comment](https://github.com/ent/contrib/pull/267#issuecomment-1077936733), I will support the hook mechanism to update the GQL schema.